docs/pages/experiments/slider.tsx

```diff
@@ -0,0 +1,331 @@
+import * as React from 'react';
+import { useTheme } from '@mui/system';
+import * as Slider from '@base_ui/react/Slider';
+import { useSliderContext } from '@base_ui/react/Slider';
+
+const axisProps = {
+  horizontal: {
+    offset: (percent: number) => ({ left: `${percent}%` }),
+    leap: (percent: number) => ({ width: `${percent}%` }),
+  },
+  'horizontal-reverse': {
+    offset: (percent: number) => ({ right: `${percent}%` }),
+    leap: (percent: number) => ({ width: `${percent}%` }),
+  },
+  vertical: {
+    offset: (percent: number) => ({ bottom: `${percent}%` }),
+    leap: (percent: number) => ({ height: `${percent}%` }),
+  },
+};
+
+export const TrackFill = React.forwardRef(function TrackFill(
+  props: any,
+  ref: React.ForwardedRef<any>,
+) {
+  // does not support inverted range fill! (yet)
```

Comment:
> "The previous track component referred to the filled portion of the slider "bar". This component is dropped, but can be easily implemented using a new useSliderContext hook" We cannot drop this feature, it's important for the UX.

It can easily be provided as a subcomponent, this here is pretty much a complete implementation, the hardest part is just choosing a name

Are you sure about `Slider.Range`? "Range" seems to imply something different as this is just the "visual fill", though I don't have any other good ideas ATM @colmtuite